keps/sig-auth/3257-trust-anchor-sets/README.md

```diff
@@ -652,6 +666,33 @@ updated to refer to the new version, and their health can then be assessed.
 Once the new version is confirmed to work, all applications can be updated to
 point to it.
 
+### Revocation
+
+While it is impossible to enforce that all clients that consume trust bundles perform
+revocation, the Kubernetes API server is well positioned to perform such checks
+since it can easily cache certificate revocation lists.
+
+When the API server parses the bundles within `pemTrustAnchors`, it will extract
+out all `CRLDistributionPoints` that have a `http://` scheme.  These CRLs
```

Comment:
Is there any plan to support a HTTPS scheme? While generally CRLs are distributed by http (This is safe as they're independently signed, though still prone to replay attacks), I've been discouraging my clients from allowing any outbound HTTP Connections from their cluster as a matter of security (not shooting themselves in the foot, really). On the other hand, introducing the HTTPS Scheme creates a whole minor issue of which trust bundle to use for that connection; Either the pemTrustAnchors or the system CA store could be appropriate in different circumstances.

Reply:
We could support https but it does complicate the code for the reasons you mentioned.  Technically we could just skip all TLS verification and rely on the same verification as we do for http.  It would be equivalently secure.

Reply:
Intentionally setting insecureSkipVerify on CRL download clients is a good call ; want to add that to the spec?

Reply:
This was contentious so we have dropped the API server bits altogether for the 1.26 release.